packages/webapp/src/components/Form/Input/index.jsx

```diff
@@ -71,6 +72,8 @@ const Input = ({
     input.current.value = '';
     onChange?.({ target: input.current });
     hookFormRegister?.onChange({ target: input.current });
+    // Manually trigger validation against the new value ''
+    trigger?.(name);
```

Comment:
I noticed that text max length error remains even if the field is cleared, so I added `trigger` to validate the field and clear the out-dated error.
(This would be a problem in details view when clearing a field in one card to clear an error, go to another card and go back to the previous card. You would see the error that was cleared once)

https://github.com/LiteFarmOrg/LiteFarm/assets/33141219/51b056a2-b2e0-4406-82c0-61373ea22b3f